plugins/microchip/switchtec-nvme-device.c

```diff
@@ -100,6 +113,61 @@ int pax_nvme_submit_admin_passthru(int fd, struct nvme_passthru_cmd *cmd)
 	return status;
 }
 
+int pax_nvme_submit_admin_passthru(int fd, struct nvme_passthru_cmd *cmd)
```

Comment:
We should have another _4k_rsp function for the new implementation, and this function just try _4k_rsp and 1k_rsp and return.